css/style.css

```
@@ -0,0 +1,505 @@
+/* Fonts */
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprum.woff) format("woff"),
+    url(../fonts/cuprum.woff2) format("woff2");
```

Первым в списке должен идти более новый формат:
```suggestion
  src: url(../fonts/cuprum.woff) format("woff2"),
    url(../fonts/cuprum.woff2) format("woff");
```